pipe/runtime/src/sections/hello_world/destination.rs

```diff
@@ -0,0 +1,78 @@
+//! HelloWorld Destination example section implementation
+//!
+//! Upon receipt of a message, this section will log "Hello, World!" + the message received. Being
+//! a destination, there's nothing written to the output stream.
+use futures::{FutureExt, Sink, Stream, StreamExt};
+use section::{Command, Section, SectionChannel, State};
+use std::future::Future;
+
+use std::pin::{pin, Pin};
+
+use crate::{
+    config::Map,
+    message::Message,
+    types::{DynSection, SectionError},
+};
+
+#[derive(Debug)]
+pub struct HelloWorld {}
+
+impl HelloWorld {
+    pub fn new() -> Self {
+        Self {}
+    }
+
+    pub async fn enter_loop<Input, Output, SectionChan>(
+        self,
+        input: Input,
+        _output: Output,
+        mut section_chan: SectionChan,
+    ) -> Result<(), SectionError>
+    where
+        Input: Stream<Item = Message> + Send + 'static,
+        Output: Sink<Message, Error = SectionError> + Send + 'static,
+        SectionChan: SectionChannel + Send + Sync + 'static,
+    {
+        let mut input = pin!(input.fuse());
+        loop {
+            futures::select! {
+                msg = input.next() => {
+                    let mut msg = match msg {
+                        Some(msg) => msg,
+                        None => Err("input stream closed")?
+                    };
+
+                    let payload = &msg.payload;
+                    let origin = &msg.origin;
+                    log::info!("Message from '{:?}' received! {:?}", origin, payload);
```

Comment:
We can use section_chan.log() function for that, it's far from perfect, but allows us to write logs with additional metadata such as section/pipe ids